src/components/LoginForm/index.js

```diff
@@ -68,14 +35,67 @@ function Login({ providers, ...props }) {
   }, [session]);
 
   return (
-    <Grid
-      container
-      justify="space-around"
-      alignItems="center"
-      className={classes.root}
-    >
-      <Grid item xs={12} className={classes.item}>
-        <form noValidate className={classes.formStyles}>
+    <Section className={classes.section}>
+      <Grid container>
+        <Grid item xs={7} className={classes.container}>
```

Comment:
Given our current design is for xl only, I think this should be md={7} or even lg={7}... Otherwise, you end up with 👇🏽 

![Screenshot 2022-01-10 at 10-52-17 Sign in TrollTracker](https://user-images.githubusercontent.com/1779590/148733248-525bcc51-d59c-46a8-9496-3ec83c50999e.png)